test-d/types/typesUtils.test-d.ts

```diff
@@ -6,3 +6,12 @@ expectType<{ a?: string }>(membersBecomeOptional);
 
 const nestedMembersBecomeOptional: DeepPartial<{ a: { b: string } }> = { a: {} };
 expectType<{ a?: { b?: string } }>(nestedMembersBecomeOptional);
+
+const expanded_1: Expanded<{ foo: string }> = { foo: '' };
```

Comment:
Personally, I'd avoid underscores. We seem to use camel case for everything, so I would not use underscores here.

Also maybe we could add a test for a more complicated object, e.g. with several different keys being expanded?